mlflow/server/js/src/components/artifact-view-components/ShowArtifactImageView.js

```diff
@@ -52,64 +63,81 @@ class ShowArtifactImageView extends Component {
     img.src = this.getSrc();
   };
 
-  render() {
+  imageContainer = ({ children }) => (
```

Comment:
Is there a specific reason to define this as a class property instead of separately as for example `GifImageContainer`?

Reply:
I wrote this function to avoid writing the following code twice and reuse it in `renderGif` and `renderStaticImage`

```html
<div className="image-outer-container">
  <div className="image-container">{...}</div>
</div>
```

Reply:
I just realized this function isn't necessary. We can put this nested `div` part in `render` like:

```javascript
render() {
  return(
    <div className="image-outer-container">
      <div className="image-container">
        {this.isGif() ? this.renderGif() : this. renderStaticImage()}
      </div>
    </div>
  )
}
```